ruby/resources/scripts/script.js

```diff
@@ -0,0 +1,101 @@
+var payButton = document.getElementById('pay-button');
+var form = document.getElementById('payment-form');
+var errorStack = [];
+
+Frames.init('pk_sbox_XXX');
+
+// Only enable the payment button when the payment form is valid
+Frames.addEventHandler(
+    Frames.Events.CARD_VALIDATION_CHANGED,
+    () => (payButton.disabled = !Frames.isCardValid())
+);
+
+// When the form is submitted intercept the
+form.addEventListener('submit', function (event) {
+    event.preventDefault();
+    Frames.submitCard();
+});
+
+// When the tokenization is completed retrieve the card token and send to the server to complete a payment
+Frames.addEventHandler(Frames.Events.CARD_TOKENIZED, (event) => {
+    // Send the token to the server to complete the payment
+    http(
+        {
+            method: 'POST',
+            route: '/payments',
+            body: {
+                token: event.token, // the card token
+            },
+        },
+        // This function is called after the server code is executed
+        (response) => {
+            let myApiResponse;
+            if (response.status >= '300') {
```

Comment:
Probably we should increase to 400, I know that 300 means different stuff than OK, but we can avoid some confusion if people thinks that 3xx is possible somehow

Reply:
We have it like these in all the SDKs samples. I'll change it here and then we can update the rest of the samples. Tracking it with this [task](https://checkout.atlassian.net/browse/INT-1168)